packages/debug/src/browser/editor/debug-exception-widget.tsx

```diff
@@ -70,15 +72,13 @@ export class DebugExceptionWidget implements Disposable {
 
     protected render(info: DebugExceptionInfo): void {
         const stackTrace = info.details && info.details.stackTrace;
-        ReactDOM.render(<React.Fragment>
-            <div className='title'>{info.id ? `Exception has occurred: ${info.id}` : 'Exception has occurred.'}</div>
-            {info.description && <div className='description'>{info.description}</div>}
-            {stackTrace && <div className='stack-trace'>{stackTrace}</div>}
-        </React.Fragment>, this.zone.containerNode, () => {
-            const lineHeight = this.editor.getControl().getOption(monaco.editor.EditorOption.lineHeight);
-            const heightInLines = Math.ceil(this.zone.containerNode.offsetHeight / lineHeight);
-            this.zone.layout(heightInLines);
-        });
+        ReactDOM.render(
+            <React.Fragment>
+                <div className='title'>{info.id ? `Exception has occurred: ${info.id}` : 'Exception has occurred.'}</div>
+                {info.description && <div className='description'>{info.description}</div>}
+                {stackTrace && <div className='stack-trace'>{stackTrace}</div>}
+            </React.Fragment>,
```

Comment:
I assume it was to resize on each rendering? Why do we need to remove it?

Reply:
That callback was actually did nothing (as far as i saw).
While this component is rendering the `containerNode.offsetHeight` is `0` (i assume because it is not rendered yet in the zoneNode) and therefore heightInLines is `0`. 
Getting the `containerNode.offsetHeight` on `onComputedHeight` after the viewZone has rendered, worked for me.
https://github.com/eclipse-theia/theia/pull/8344/files#diff-e36030c15530769fcd2fe88693fd3cedR139

Reply:
ok, it seems react semantic has changed and it is matter of rendering exception after zone